lib/matplotlib/tests/test_quiver.py

```diff
@@ -277,3 +278,19 @@ def test_quiver_setuvc_numbers():
 
     q = ax.quiver(X, Y, U, V)
     q.set_UVC(0, 1)
+
+
+@check_figures_equal()
+def test_zorder(fig_test, fig_ref):
+    """Check QuiverKey zorder option"""
+    X, Y, U, V = 1, 1, 2, 2
+
+    ax_test = fig_test.subplots()
+    zorder_value = 5.
```

Comment:
The default `zorder` for `quiver` is 1, so the default for `quiverkey` is 1.1. That means `zorder=5` may effectively be the same as not specifying anything? So I think this test needs at least another artist in between 1.1 and 5 to confirm that the key really is moved up.